.eslintignore

```diff
@@ -1,6 +1,7 @@
 # Libs we don't need to check
 node_modules
 **/vendor
+lib
```

Comment:
We can lint the run/util files. I fixed this c6f5a494943838d2a746a1f2dae25319ceae4835.